Dockerfile.backwards

```
@@ -7,8 +7,7 @@ RUN echo "yaml https://s3-us-west-2.amazonaws.com/rosdep/base.yaml" > /etc/ros/r
 COPY . /opt/package
 WORKDIR /opt/package
 
-RUN apt-get install curl
-RUN curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | sudo apt-key add -
+RUN apt update || apt install -y curl && curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
```

```suggestion
RUN apt-get update && apt-get install --no-install-recommends --yes curl
RUN curl --fail --silent --show-error --location https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
```

[Output of `apt` is designed for end-users, so its parameters and output are not stable. Use `apt-get` in scripts instead](http://manpages.ubuntu.com/manpages/bionic/man8/apt.8.html#script%20usage%20and%20differences%20from%20other%20apt%20tools)